src/encord_active/server/routers/project2_tags.py

```diff
@@ -52,15 +62,29 @@ def route_list_tags(project_hash: uuid.UUID, engine: Engine = Depends(dep_engine
         ).fetchall()
     return [ProjectTagEntry(hash=tag_hash, name=name) for tag_hash, name in tags]
 
+@router.get("/meta")
+def route_list_tags(project_hash: uuid.UUID, engine: Engine = Depends(dep_engine_readonly)) -> List[ProjectTagEntryMeta]:
```

Comment:
tag meta is expensive - so only run against all tags that this is actually needed for. & this is a name clash on the routes.

route should be similar to:

```
@router.get("/{tag_hash}/meta")
 route_tag_meta(project_hash: uuid.UUID, tag_hash: uuid.UUID, engine: Engine = ...) -> TagMetadata:
```

Reply:
`.where(ProjectTag.project_hash == project_hash)`

It is only running against tags needed. We need to show all tags for a project in the new collections tab.